docs/developers/nft-tokens.md

```diff
@@ -196,6 +233,14 @@ NFTCreationTransaction {
 }
 ```
 
+Additional gas refers to:
+- Transaction payload cost: Data field length * 1500 (GasPerDataByte = 1500)
+- Storage cost: Size of NFT data * 50000 (StorePerByte = 50000)
+
+:::tip
+Please note that because NFTs are stored in accounts trie, every transaction involving the NFT will require higher gas depending on NFT data size.
```

Comment:
No need to say "please" in technical documentation :)

```suggestion
Note that because NFTs are stored in accounts trie, every transaction involving the NFT will require a gas limit depending on NFT data size.
```